src/app/components/Footer/index.tsx

```diff
@@ -209,4 +213,148 @@ const Footer = ({
   </Styled.FooterWrapper>
 );
 
+// tslint:disable-next-line:variable-name
+const PortalColumn1 = () => (
+  <Styled.Column1>
+    <Copyrights values={getValues()} />
+  </Styled.Column1>
+);
+
+// tslint:disable-next-line:variable-name
+export function ContactDialog({
+  isOpen,
+  close,
+  contactFormParams,
+  className,
+}: {
+  isOpen: boolean;
+  close: () => void;
+  contactFormParams?: {key: string; value: string}[];
+  className?: string;
+}) {
+  const contactFormUrl = React.useMemo(() => {
+    const formUrl = '/embedded/contact';
+
+    if (contactFormParams !== undefined) {
+      const params = contactFormParams
+        .map(({key, value}) => encodeURIComponent(`${key}=${value}`))
+        .map((p) => `body=${p}`)
+        .join('&');
+
+      return `${formUrl}?${params}`;
+    }
+
+    return formUrl;
+  }, [contactFormParams]);
+  return !isOpen ? null : (
+    <Styled.ContactDialog className={className} onModalClose={close} heading='i18n:footer:column1:contact-us'>
+      <iframe id='contact-us' title='contact-us' src={contactFormUrl} />
+    </Styled.ContactDialog>
+  );
+}
+
+export function useContactDialog() {
+  const [isOpen, setIsOpen] = React.useState(false);
+  const open = React.useCallback(() => setIsOpen(true), [setIsOpen]);
+  const close = React.useCallback(() => setIsOpen(false), [setIsOpen]);
+
+  React.useEffect(
+    () => {
+      if (typeof window === 'undefined' || !window.parent || window.parent === window) {
```

Comment:
why are you checking `!window.parent || window.parent === window` here? this looks like a check to see if the rex page is embedded, thats not what we want right?